### ISSUE TITLE
Missing a "}" in Navigation Menu Button Example

### ISSUE THREAD

**Issue by fengdetiankong6** [w3c/aria-practices#1203]
In the [Navigation Menu Button Example ](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html) page, it lacks of a "}" in the HTML Source Code. 
![image](https://user-images.githubusercontent.com/29356851/66361806-d5af5980-e9b2-11e9-934e-938654a88d4d.png)

**Comment by nschonni** [comment]
Looks like it is correct in the source
https://github.com/w3c/aria-practices/blob/545d85226ac700119e76634d62a76d69b0117cf6/examples/menu-button/menu-button-links.html#L99-L102
That section hasn't changed in 2 years looking at the blame, so the w3c must be running an old version

**Comment by nschonni** [comment]
Nevermind, the source is correct, something is going wrong in how the "HTML Source Code" section is being generated

**Comment by fengdetiankong6** [comment]
Yes, there is something wrong in how the "HTML Source Code" section being generated. Uh, so is it nessecary to solve it? It is in the example.js.

**Comment by ZoeBijl** [comment]
Have marked as bug in source code script.

**Comment by jnurthen** [comment]
fyi I was debugging but ran out of time. The final `}` appears to get removed in the stripIndentation function in examples.js

**Comment by ZoeBijl** [comment]
For anyone interested in looking at this. You can find the functionality in https://github.com/w3c/aria-practices/blob/master/examples/js/examples.js

**Comment by fengdetiankong6** [comment]
` for (var i = 0; i < node.childNodes.length; i++) {
    var childNode = node.childNodes[i];

    switch (childNode.nodeType) {
      case Node.ELEMENT_NODE:
        this.createCode(sourceCodeNode, childNode, indentLevel, false);
        break;

      case Node.TEXT_NODE:
        if (hasText(childNode.nodeValue)) {
          var textNodeContent = sanitizeNodeValue(childNode.nodeValue).trim();
          textNodeContent = indentLines(textNodeContent, indentation(indentLevel));

          sourceCodeNode.innerHTML += '<br/>' + textNodeContent;
        }
        break;

      case Node.COMMENT_NODE:
        if (hasText(childNode.nodeValue)) {
          var commentNodeContent = sanitizeNodeValue(childNode.nodeValue);
          commentNodeContent = '&lt;!--' + commentNodeContent + '--&gt;';
          commentNodeContent = indentLines(commentNodeContent, indentation(indentLevel));

          sourceCodeNode.innerHTML += '<br/>' + commentNodeContent;
        }
        break;
    }
  }`
It seems that the trim() method leads to the issue.

**Comment by fengdetiankong6** [comment]
I have found the functionality in [examples/js/examples.js](https://github.com/w3c/aria-practices/blob/master/examples/js/examples.js). The reason of it is that there is a indent near the '}', deleting it is ok.
